src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/ProcessFullyApprovedCohort/ProcessFullyApprovedCohortCommandHandler.cs

```diff
@@ -76,7 +76,9 @@ protected override async Task Handle(ProcessFullyApprovedCohortCommand request,
                         })
                         .ToArray(),
                     ContinuationOfId = a.ContinuationOfId,
-                    DateOfBirth = a.DateOfBirth.Value
+                    DateOfBirth = a.DateOfBirth.Value,
+                    ActualStartDate = a.ActualStartDate,
+                    IsOnFlexiPaymentPilot = a.IsOnFlexiPaymentPilot
```

Comment:
There is a line before this

`StartDate = a.StartDate.Value,`

Needs to have .Value removed otherwise it'll continue to throw exceptions for pilot learners.

Reply:
ah good spot